web/src/installers/index.ts

```diff
@@ -812,7 +812,7 @@ export class Installer {
 
     i.id = "latest";
     i.spec.kubernetes = { version: "1.23.x" };
-    i.spec.containerd = { version: this.toDotXVersion(installerVersions.containerd[0]) };
+    i.spec.containerd = { version: "1.5.x" }; // 1.6.x is not compatible with Weave
```

Comment:
I would prefer to mark containerd 1.6.x as NOT latest, so that we don't have people attempting to upgrade run into it